src/common/OneDSLoggerTelemetry/oneDSLogger.ts

```diff
@@ -342,8 +342,15 @@ export class OneDSLogger implements ITelemetryLogger{
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     private populateVscodeDesktopAttributes(envelope: any){
-        // TODO: this method helps in populating desktop attributes.
-        console.log(envelope);
+        if (envelope.data.eventName == desktopExtTelemetryEventNames.DESKTOP_EXTENSION_INIT_CONTEXT) {
+            OneDSLogger.contextInfo.orgId = JSON.parse(envelope.data.eventInfo).OrgId;
+            OneDSLogger.contextInfo.envId = JSON.parse(envelope.data.eventInfo).EnvironmentId;
+            // TODO: Populate website id
+            OneDSLogger.contextInfo.websiteId = 'test'
+        }
+        envelope.data.measurements = envelope.data.measurement;
+        envelope.data.timestamp = envelope.time;
```

Comment:
Why this is in this method? It should be at the parent method i.e. populateCommonAttributes

Reply:
websiteId will be fetched in a way very specific to desktop extension so it's in this method.
For other 2 attributes - measurements & timestamp - they need to be populated for desktop extension so added here.
Moving these to common would lead to them being populated the same way. Is the envelope & data returned inside it for web follow similar structure?

Reply:
Moved measurements & timestamp to common attributes method as they are needed for web too.